modules/distr-tracing-tempo-deploy-production-es.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
```

Comment:
For Tempo we don't have multiple deployment strategies. We can remove this file.

Reply:
Outdated. Resolving this thread.